spec/basic/expected_conditions_spec.js

```diff
@@ -1,255 +1,250 @@
-var EC = protractor.ExpectedConditions;
+const EC = protractor.ExpectedConditions;
 
-describe('expected conditions', function() {
-  beforeEach(function() {
-    browser.get('index.html#/form');
+describe('expected conditions', () => {
+  beforeEach(async () => {
+    await browser.get('index.html#/form');
   });
 
-  it('should have alertIsPresent', function() {
-    var alertIsPresent = EC.alertIsPresent();
+  it('should have alertIsPresent', async () => {
+    const alertIsPresent = await EC.alertIsPresent();
```

Comment:
EC.alertIsPresent() returns a function and not a promise.

https://github.com/angular/protractor/blob/master/lib/expectedConditions.ts#L144-L171

So the fix for this would be:

```
const alertIsPresent = EC.alertIsPresent();
expect(await alertIsPresent.call()).toBe(false);  
```